framework/wazuh/exception.py

```diff
@@ -190,7 +194,31 @@ class WazuhException(Exception):
         3020: 'Timeout sending request',
         3021: 'Timeout executing API request',
         3022: 'Unknown node ID',
-        3023: 'Worker node is not connected to master'
+        3023: {'message': 'Worker node is not connected to master',
+               'remediation': 'Check the cluster.log file to see if there are connection errors. '
```

Comment:
We should tell the user where the cluster.log file is (WAZUH_HOME/logs/cluster.log)